lib/internal/process/pre_execution.js

```diff
@@ -107,7 +107,6 @@ function prepareExecution(options) {
     // (including preload modules).
     initializeClusterIPC();
 
-    // TODO(joyeecheung): do this for worker threads as well.
     require('internal/v8/startup_snapshot').runDeserializeCallbacks();
```

Comment:
The worker thread also runs this function, you can just move this outside the block then it gets run by both.

For now though this isn't really meaningful because there aren't any deserialize callback added for the workers.

Reply:
I see so closing the PR for now 
Thanks!